README.md

```diff
@@ -1,2 +1,48 @@
 # system_uri
+
 Desktop System App URI registration handler 
+
```

Comment:
You need to copy the line for maintainership: https://raw.githubusercontent.com/maidsafe/safe_core/3a38924eb8a23a9c6ba82ad02834aaee2da30865/README.md